usaspending_api/awards/migrations/0077_auto_20170405_2102.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-04-05 21:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.models import F
+
+
+def forwards_or_backwards_func(apps, schema_editor):
+    """
+    Because -1 * -1 == 1, this migration works the same both directions!
+    """
+    db_alias = schema_editor.connection.alias
+
+    cls = apps.get_model('awards', 'FinancialAccountsByAwards')
+
+    # invert values of all fields ending in _cpe or _fyb
+    updates = {field_name: F(field_name) * -1
+               for field_name in dir(cls) if (
+                   field_name.endswith('_cpe') or field_name.endswith('_fyb'))}
+    cls.objects.using(db_alias).update(**updates)
+
+    cls.objects.using(db_alias).update(
+        transaction_obligated_amount=F('transaction_obligated_amount') * -1)
+
```

Comment:
Since we've already loaded the quarterly numbers for file B, this migration will need to reload those to ensure that they are calculated using the corrected signs. Will make this change and ping @afrasier for a look.